### PR TITLE
Update .ghal.rules.json for new labeler

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

Depends on the new labeler version.

- [x] New labeler version is deployed
- New rule for **Issue** which detects the `:checkered_flag: Release .NET 6` label and adds it to [.NET 6 documentation plan (132)](https://github.com/dotnet/docs/projects/132)
- New rule for **PR** which detects the `:checkered_flag: Release .NET 6` label and adds it to [.NET 6 documentation plan (132)](https://github.com/dotnet/docs/projects/132)
- Remove WinForms/WPF file detection paths

### CODE REVIEW

File: .ghal.rules.json [hunk 1]
```diff
@@ -320,6 +327,15 @@
           }
         }
       }
+    },
+    "labeled": {
+      "processor-conditional-release6": {
+        "type": "query",
+        "value": "length(Issue.labels[?name == ':checkered_flag: Release .NET 6']) != `0`",
```
Comment (gewarren) [inline .ghal.rules.json:334]: Should this be Issue.labels or something more like PullRequest.labels?

Also, we can probably remove the label sections for framework/winforms and framework/wpf, right? (I think those folders have been removed.)

Comment (adegeo) [reply .ghal.rules.json:334]: Thanks! Great copy/paste catch 😁

**Comment by adegeo** [comment]
@gewarren If you're creating a breaking changes project, I can add that rule too

**Comment by gewarren** [comment]
> @gewarren If you're creating a breaking changes project, I can add that rule too

Thank you. It is project #135.

**Review by gewarren — APPROVED** [review]
Thank you!

**Pull request merged**